Source/SpatialGDK/Public/SpatialInterop.h

```diff
@@ -93,10 +94,12 @@ using FOutgoingPendingArrayUpdateMap = TMap<const UObject*, FChannelToHandleToOP
 
 using FResolvedObjects = TArray<TPair<UObject*, const improbable::unreal::UnrealObjectRef>>;
 
-// Helper function to write incoming replicated property data to an object.
-FORCEINLINE void ApplyIncomingReplicatedPropertyUpdate(const FRepHandleData& RepHandleData, UObject* Object, const void* Value, TSet<UProperty*>& RepNotifies)
+// Helper function to write incoming replicated property data to an object
+// TargetObject is the actor or actor component that owns the replicated property and its RepNotifies, NOT the root actor channel
```

Comment:
"NOT the root actor channel" is misleading. If the property is on the actor, the `TargetObject` will in fact be the channel actor. I think the first part already explains it well, so just remove the "NOT the root actor channel" part.